vm/devices/storage/storage_tests/src/tests/scsidvd_nvme.rs

```diff
@@ -0,0 +1,227 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
This file moves into `storage_tests/tests`. Delete the `src` directory altogether.